.eslintrc

```diff
@@ -51,6 +51,7 @@
     "no-restricted-syntax": 0,
     "no-prototype-builtins": 0,
     "no-void": 0,
+    "no-lonely-if": 0,
```

Comment:
I totally understand why disabling for `logger.js` if/else blocks :) , but that rule is pretty neat I think. What about disabling it locally for particular use cases?